CHANGELOG.md

```diff
@@ -19,7 +20,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
  - Define a different output file for each sourceSet
 ### Fixed
- - Fixed plugin doesn't apply custom reporter for ktlint versions >0.10.x (#28)
+- Fixed plugin doesn't apply custom reporter for ktlint versions >0.10.x (#28)
```

Comment:
Unnecessary whitespace change?

Reply:
Oh, damn, will update

Reply:
Check this commit d26f377041144ac55b188d5dcc475da472ebb1a8